src/index.html

```diff
@@ -11,8 +11,67 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wdth,wght@0,75..100,100..900;1,75..100,100..900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div>
+        <img
+          src="../images/imac.jpeg"
+          alt="imac.jpeg"
+          class="card__image"
+        />
+      </div>
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
```

Comment:
Need to use the semantic tag for text, fix it everywhere
```suggestion
      <h3 class="card__title">
        APPLE A1419 iMac 27" Retina
        <br />
        5K Monoblock (MNED2UA/A)
      </h3>

      <p class="card__product-code">Product code: 195434</p>
```